syntax = "proto3";

package fraud_detection;

service fraud_detection {
    rpc fraud_detect (f_detectionRequest) returns (f_detectionResponse);
}

message f_detectionRequest {
    string name = 1;
    string id =2 ;
}

message f_detectionResponse {
    bool is_fraud = 1;
}
